src/config.rs

```diff
@@ -169,6 +174,9 @@ pub struct StratusConfig {
     #[clap(flatten)]
     pub miner: MinerConfig,
 
+    #[clap(flatten)]
+    pub importer: ImporterConfig,
```

Comment:
`ImporterConfig` should be optional instead of having its inner values as optional.
If not specified, the consensus module should not be initialized.